README.md

```diff
@@ -11,6 +11,13 @@ metadata:
     kubernetes.io/tls-acme: "true"
 ```
 
+### Configure ACME
```

Comment:
It would be better to just reference a file with advanced options here using a link
also there already is https://github.com/tnozicka/openshift-acme/blame/master/docs/design/architecture.adoc#L26